src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimize.cpp

```diff
@@ -92,12 +92,16 @@ QMCFixedSampleLinearOptimize::QMCFixedSampleLinearOptimize(MCWalkerConfiguration
       Max_iterations(1),
       wfNode(NULL),
       param_tol(1e-4),
-      generate_samples_timer_(
-          *timer_manager.createTimer("QMCLinearOptimizeBatched::GenerateSamples", timer_level_medium)),
-      initialize_timer_(*timer_manager.createTimer("QMCLinearOptimizeBatched::Initialize", timer_level_medium)),
-      eigenvalue_timer_(*timer_manager.createTimer("QMCLinearOptimizeBatched::Eigenvalue", timer_level_medium)),
-      line_min_timer_(*timer_manager.createTimer("QMCLinearOptimizeBatched::Line_Minimization", timer_level_medium)),
-      cost_function_timer_(*timer_manager.createTimer("QMCLinearOptimizeBatched::CostFunction", timer_level_medium))
+      generate_samples_timer_(*timer_manager.createTimer("QMCLinearOptimize::GenerateSamples", timer_level_medium)),
+      initialize_timer_(*timer_manager.createTimer("QMCLinearOptimize::Initialize", timer_level_medium)),
+      derivative_timer_(
+          *timer_manager.createTimer("QMCLinearOptimize::optTarget->checkConfigurations()", timer_level_medium)),
+      buildmat_timer_(*timer_manager.createTimer("QMCLinearOptimize::optTarget->fillOverlapHamiltonianMatrices()",
+                                                 timer_level_medium)),
+      eigenvalue_timer_(*timer_manager.createTimer("QMCLinearOptimize::EigenvalueSolve", timer_level_medium)),
+      invert_matrix_timer_(*timer_manager.createTimer("QMCLinearOptimize::invert_matrix", timer_level_medium)),
```

Comment:
InvertOverlapMat is a name better than invert_matrix

Reply:
Yes.. all the timer names need to be verbose and not generic.

Reply:
Renamed.